Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -1620,67 +1660,27 @@ public void Save()
         /// </summary>
         public void UpdateResultView(ICollection<ResultsForUpdate> resultsForUpdates)
         {
-            if (!resultsForUpdates.Any())
+            if (resultsForUpdates == null || !resultsForUpdates.Any())
                 return;
-            CancellationToken token;
 
+            // Block the query execution when Open ContextMenu is called
+            if (!QueryResultsSelected())
```

Comment:
I don't understand here. Query shouldn't be executed when context menu is called? It will only call QueryContextMenu?

Reply:
I already mentioned this in the PR description. If you open the context menu while results are still loading, both the context menu and the results panel end up being shown once the loading finishes. Please read the PR description. Why do you always ignore what others have already said? I always write it down, whether it's on Discord or here, but you keep asking the same questions. You’re making me explain the same thing over and over again.